setup.py

```diff
@@ -2,10 +2,12 @@
 from setuptools import setup
 
 setup(
-    name='pip-install-privates',
```

Comment:
are you going to rename the package aswell?

Reply:
the package name is already 'pipprivates' and this name has to match that afaik. perhaps we should change the package name to pip-install-privates instead. if the package is named pipprivates the repo name doesn't match anymore which is also confusing. perhaps it would even be better to keep the package name pipprivates and then also rename the repo to pipprivates as well.

Reply:
If you change to `pip-install-privates` I think the folder name should be `pip_install_privates` with underscore. 
The two choices are good either. but maybe pip install privates is more explicit than pipprivates

Reply:
:+1: